peakina/io/ftp/ftp_utils.py

```diff
@@ -16,6 +16,8 @@
 import paramiko
 
 FTP_SCHEMES = ["ftp", "ftps", "sftp"]
+_DEFAULT_MAX_TIMEOUT = 10
```

Comment:
tu peux spécifier l'unité dans la constante?
```suggestion
_DEFAULT_MAX_TIMEOUT_SECONDS = 10
```

Reply:
https://github.com/toucantoco/peakina/pull/600/commits/e4e4a5d